slides.md

```diff
@@ -605,9 +606,9 @@ export default async function ecommerce(fastify) {
 <div class="dense">
 
 - Run the server for step 5 (`cd src/a05-security-misconfiguration`, `npm start`)
-- In Postman, run the query for `A05: Login`. Observe a cookie with `userId=1` being returned
-- Try to run the query for `A05: Profile`. Observe the information about profile with `userId=1` being returned
-- Try to [change the value of the cookie](https://learning.postman.com/docs/sending-requests/cookies/) to `userId=2`. Observe information about `userId=2` being returned
+- From the browser at `http://localhost:3006`, run the query for `A05: Login`. Observe a cookie with `userId=1` being returned
```

Comment:
Exercise 5 is not what it should be, it should be about manipulating cookies, probably copy paste error:
<img width="1920" alt="Screenshot 2023-10-16 at 12 33 35" src="https://github.com/nearform/owasp-top-ten-workshop/assets/36735501/3ce670ad-827b-441b-9ae6-5c6b4983e54c">